src/obis-observer/commands/uplink/GetMeterInfoResponse.ts

```diff
@@ -0,0 +1,100 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+
+
+/**
+ * IGetMeterInfoResponseParameters command parameters
+ */
+interface IGetMeterInfoResponseParameters extends ICommandParameters {
+    meterProfileId: number,
+    address: string
+}
+
+
+const COMMAND_ID = 0x79;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'get meter info response',
+        parameters: {
+            requestId: 2,
+            meterProfileId: 1,
```

Comment:
if profile not assigned to meter - here will be 0?

Reply:
What is more correct?
- make meterProfileId optional;
- remove this command, and use GetMeterAddress, GetMeterProfile instead;
- provide INVALID_METER_PROFILE_ID and INVALID_METER_ID const values (which i have on the observer)

Reply:
make meterProfileId optional - ok

Reply:
did it